graphgym/contrib/loader/roland_reddit.py

```diff
@@ -0,0 +1,174 @@
+import os
```

Comment:
We have merged `roland_xx` to a single file called `roland`, so I think this should be removed?